compiler/rustc_parse_format/src/lib.rs

```diff
@@ -925,6 +925,18 @@ impl<'a> Parser<'a> {
                             suggestion: Suggestion::UsePositional,
                         },
                     );
+                } else if let ArgumentIs(_) = field.position {
```

Comment:
Can you make this into an exhaustive match on `field.position` rather than two `if let`s?

Reply:
Changed it to use match.